README.md

```diff
@@ -1,11 +1,11 @@
 # AWESOME BOOKS
 
-> In this project, I will build an application that allows users to create their own transaction lists. In this application, users can also create groups with which they can add their transactions. See live demo........
-
-## Explanation video
+> "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
 
```

Comment:
**Remember to keep the general requirements on mind it's mandatory on each project**
- Kindly enable your ``CSS linters`` and fix issues. after that request another code review.